nvflare/app_common/pt/pt_file_model_persistor.py

```diff
@@ -226,3 +226,38 @@ def save_model(self, ml: ModelLearnable, fl_ctx: FLContext):
         assert isinstance(self.persistence_manager, PTModelPersistenceFormatManager)
         self.persistence_manager.update(ml)
         self.save_model_file(self._ckpt_save_path)
+
+    def _load_global_model(self, model_file, fl_ctx: FLContext) -> ModelDescriptor:
+        try:
+            # device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+            # Use the "cpu" to load the global model weights, avoid GPU out of memory
+            device = "cpu"
+            location = os.path.join(self.log_dir, model_file)
+            data = torch.load(location, map_location=device)
+            persistence_manager = PTModelPersistenceFormatManager(data, default_train_conf=self.default_train_conf)
+            learnable = persistence_manager.to_model_learnable(self.exclude_vars)
+            model_descriptor = ModelDescriptor(
+                name=self.global_model_file_name,
+                location=location,
+                model_format=ModelDescriptor.PT_CHECKPOINT,
```

Comment:
It should be the model persistence manager that decides the model format.
We should add a method to the persistence manager to return model format, and we call that method here.